dashboard/app/views/layouts/application.html.haml

```diff
@@ -46,6 +46,7 @@
     = render inline: File.read(Rails.root.join('..', 'shared', 'haml', 'answerdash.haml')), type: :haml if view_options[:answerdash]
     = javascript_include_tag 'application', 'data-ot-ignore' => ''
     %script{src: webpack_asset_path("js/#{js_locale}/common_locale.js"), 'data-ot-ignore' => ''}
+    %script{src: webpack_asset_path("js/#{js_locale}/signup_locale.js"), 'data-ot-ignore' => ''}
```

Comment:
@mgc1194 we actually had that other line before- it was this one we were missing!

Reply:
For the record, we were facing the following `TypeError` that pointed to some webpack configuration missing.
```
TypeError
_locale.default.create_your_free_account is not a function
Call Stack
 LoginTypeSelection
  webpack-internal:///7349:22:98
 renderWithHooks
  webpack-internal:///731:14985:18
 updateFunctionComponent
  webpack-internal:///731:17356:20
 beginWork
  webpack-internal:///731:19063:16
 HTMLUnknownElement.callCallback
  webpack-internal:///731:3945:14
 Object.invokeGuardedCallbackDev
  webpack-internal:///731:3994:16
 invokeGuardedCallback
  webpack-internal:///731:4056:31
 beginWork$1
  webpack-internal:///731:23959:7
 performUnitOfWork
  webpack-internal:///731:22771:12
 workLoopSync
  webpack-internal:///731:22702:5
```
I have added this PR as a reference in the ['work-in-progress' i18n guide](https://docs.google.com/document/d/1grBnj5IXeqg1hSsWdE-rM85FDiQmASxtvAlcv5jfk18/edit)